bot/data/tag_repository.py

```diff
@@ -31,6 +31,8 @@ async def check_tag_exists(self, name: str, guild_id) -> bool:
 
     async def get_tag_content(self, name: str, guild_id: int) -> str:
         async with aiosqlite.connect(self.resolved_db_path) as db:
+            await db.execute('UPDATE Tags SET useCount = useCount + 1 WHERE name = ? AND fk_guildId = ?', (name, guild_id,))
```

Comment:
this should be a separate query function, get_tag_function implies that you are only getting, incrementing a database field is unexpected here and should be its own function, this will also incremeent everytime !info is called on a tag

Reply:
Ok, ill abstract that to another function tomorrow.
But the info command doesnt call it from what i got in my tests. but will be changed regardless.

Reply:
Fixed